kevm-pyk/src/kevm_pyk/utils.py

```diff
@@ -347,3 +347,53 @@ def legacy_explore(
                 id=id,
                 trace_rewrites=trace_rewrites,
             )
+
+
+def escaped_chars() -> list[str]:
```

Comment:
`__` make the function private.

Reply:
I've done it a `@classmethod`, is it good enough ?

Reply:
I'm not that familiar with Python. This is just a suggestion and is not important. Although documentation would be really nice.

Reply:
Will do, thanks!